### DISCUSSION TITLE
在哪里可以设置不需要登陆就能看图片吗、

### DISCUSSION THREAD

**Discussion by tfnhui** [photoprism/photoprism#3613] [General]
在哪里可以设置不需要登陆就能看图片吗、

**Comment by graciousgrey** [comment]
You can use the following config variable

[PHOTOPRISM_AUTH_MODE: public](https://docs.photoprism.app/getting-started/config-options/#authentication)

Kindly note that PHOTOPRISM_AUTH_MODE: public  is NOT recommended for instances installed on a server accessible from outside your home network.
